sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -16,7 +16,7 @@ import { getString, getStringOrUndefined } from "../util/utils";
  * Builds the rule object from the raw json object gotten after deserializing the
  * response from the service
  */
-export function buildRule(rawRule: any): RuleProperties {
+export function buildRule(rawRule: Record<string, any>): RuleProperties {
```

Comment:
Why "any", why not "unknown"?

Reply:
In the following lines we try to access the different properties using the property name which we cannot do on `unknown`. Ideally, we should have a type for what the rawRule would be and use that